src/main/java/com/github/onsdigital/logging/builder/LogMessageBuilder.java

```diff
@@ -92,9 +89,6 @@ public void log() {
 
         logThreadPool.submit(() -> {
             if (LOG == null || !StringUtils.equalsIgnoreCase(LOG.getName(), getLoggerName())) {
-                LoggerContext context = (LoggerContext)LoggerFactory.getILoggerFactory();
-                StatusPrinter.print(context);
-
                 LOG = getLogger(getLoggerName());
             }
             if (this.contextMap != null) {
```

Comment:
A weird question, but why do we get a copy of the context map only to set it here?